mds/access_control/authenticate.py

```diff
@@ -0,0 +1,100 @@
+from typing import Set, Optional, List, Dict
+
+import jwt
+from django.utils.translation import ugettext as _
+from rest_framework import exceptions
+
+from .auth_means import BaseAuthMean
+from .jwt_decode import jwt_multi_decode
+from .scopes import SCOPE_ADMIN
+
+
+class RemoteUser:
+    """
+    Inspired by django.contrib.auth.models.AnonymousUser
+    """
```

Comment:
Why not subclass `django.contrib.auth` `AbstractUser` ? Many Django plugins or middlewares will expect an `AbstractUser` in their request.

Reply:
AbstractUser is a django model. Other "remote user" implementations (including the ook one in bluetils) use a standalone class for the user